R/UK.R

```diff
@@ -196,15 +196,15 @@ UK <- R6::R6Class("UK", # rename to country name
       }
     },
 
-    #' @description Initalize the UK Class
+    #' @description Initalize the UK Class `DataClass``
     #' @export
     #' @param nhsregions Return subnational English regions using NHS region
```

Comment:
Maybe not really or this PR but all the docs need to say what the type is. Here `nhsregions` is a bit of an enigma!

Reply:
I will pick this up in #280